README.md

```diff
@@ -71,6 +71,17 @@ public class PlaySpringDIConfiguration {
 }
 ```
 
+### For Spring Boot support (Optional)
+
+In order to support Spring Boot auto configurations, you can define additional context. This boot context is set as the parent context to the above play context. 
```

Comment:
Capitalize "play". 

Can also be good to link to Spring Boot auto configuration here.